tests/core/test_model.py

```diff
@@ -4445,3 +4457,43 @@ def test_incremental_by_partition(sushi_context, assert_exp_eq):
             """
         )
         load_sql_based_model(expressions)
+
+
+@pytest.mark.parametrize(
+    ["model_def", "path", "expected_name"],
+    [
+        [
+            """dialect duckdb,""",
```

Comment:
Can't we get rid of `model_def` altogether? This doesn't look necessary for the test. The `dialect` property can be left unspecified.

Reply:
Nvm, ignore this